src/main/java/gov/usds/case_issues/config/SecurityConfig.java

```diff
@@ -48,16 +62,46 @@ protected void configure(HttpSecurity http) throws Exception {
 				.httpStrictTransportSecurity().and()
 				.and()
 			.authorizeRequests()
-				.antMatchers("/actuator/health", "/health")
+				.antMatchers("/actuator/health", "/health", "/clientLogin")
 					.permitAll()
 				.anyRequest()
 					.authenticated()
 				.and()
 			.csrf()
 				.ignoringRequestMatchers(AnyRequestMatcher.INSTANCE)
+				.and()
+			.exceptionHandling()
+				.accessDeniedHandler(new CustomAccessDeniedHandler())
+				.and()
+			.exceptionHandling()
```

Comment:
Silly nit: I think the `.and().exceptionHandling()` is a no-op in this case (you can just do the `authenticationEntryPoint` call on the return value of `accessDeniedHandler`)